Api.js

```diff
@@ -14,7 +14,7 @@ function fetchAllArticles(topic = "", sortBy, order) {
   }
   return api.get(urlString).then((response) => {
     return response.data.articles;
```

Comment:
This way to deal with the queries is totally fine, but axios has its own way by passing an object as the second argument, which may make it a bit easier. e.g. `api.get("articles", { params: {topic, sortBy, order} })`